helpers/chains.py

```diff
@@ -75,6 +75,57 @@ def response_400(date=None, connection=None):
                                    date=date, body='')
     return resp
 
+def request(method, uri):
+    common_req_headers = [
```

Comment:
The comment isn't about this change, but about the whole file. What is a message chain? Why seems the core routine is called `base()` (`chain.base()` seems just sense less). Comments like
```python
    # response HTTP code
    common_resp_code = 200
    # response Date: header                                                           
    common_resp_date = date                                                           
    # response body
    common_resp_body = ''
    # common part of response headers 
    common_resp_headers = [                                                           
        'Connection: keep-alive',                                                     
    ]   
```
are obvious and don't make any sense.

Please make cleanups for the code: renames, comments, whatever.